quicktests/overlaying/tests/realistic/baseball.js

```diff
@@ -6,7 +6,7 @@ function makeData() {
 function run(svg, data, Plottable) {
   "use strict";
 
-  d3.csv("../overlaying/tests/baseball.csv").get(function(error, rows) {
+  d3.csv("/quicktests/overlaying/data/baseball.csv").get(function(error, rows) {
```

Comment:
It does indeed. Which is fine. We basically changed from a relative path (bad) to an absolute one. We run grunt on the root folder anyway